scripts/build-bundle

```diff
@@ -225,14 +273,31 @@ for i in "${!repositories[@]}"; do
   echo "${repositories[$i]}:${tags[$i]}">>${image_list_file}
 done
 
+add_addon_image() {
+  # $1  ENV name  $2 ENV
+  if [ -z $2 ]; then
+     echo "expected version environment $1 is not set"
+     exit 1
+  else
+    echo $2 >>  ${image_list_file}
+  fi
+}
+
+add_addon_images_to_list() {
+  # vm-import-controller: get images from values.yaml
```

Comment:
```suggestion
```

maybe we don't need the comment anymore. @w13915984028 can you help add one comment to say these variables are from the file

`source ${addons_path}/version_info`

It will help people understand the code.

Reply:
Updated the comment, thanks.